python/delta/integration_tests/unity-catalog-commit-coordinator-integration-tests.py

```diff
@@ -54,54 +63,221 @@
     .config(f"spark.sql.catalog.{CATALOG_NAME}.token", CATALOG_TOKEN) \
     .config(f"spark.sql.catalog.{CATALOG_NAME}.uri", CATALOG_URI) \
     .config(f"spark.sql.defaultCatalog", CATALOG_NAME) \
+    .config("spark.databricks.delta.replaceWhere.constraintCheck.enabled", True) \
     .config("spark.hadoop.fs.s3.impl", "org.apache.hadoop.fs.s3a.S3AFileSystem") \
     .config("spark.databricks.delta.commitcoordinator.unity-catalog.impl",
             "org.delta.catalog.UCCoordinatedCommitClient") \
     .getOrCreate()
 
-expected_error_tag = "UNITY_CATALOG_EXTERNAL_COORDINATED_COMMITS_REQUEST_DENIED"
+MANAGED_CATALOG_OWNED_TABLE_FULL_PATH = f"{CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE}"
+MANAGED_NON_CATALOG_OWNED_TABLE_FULL_PATH = f"{CATALOG_NAME}.{SCHEMA}.{MANAGED_NON_CC_TABLE}"
```

Comment:
```suggestion
MANAGED_CATALOG_OWNED_TABLE_FULL_NAME = f"{CATALOG_NAME}.{SCHEMA}.{MANAGED_CC_TABLE}"
MANAGED_NON_CATALOG_OWNED_TABLE_FULL_NAME = f"{CATALOG_NAME}.{SCHEMA}.{MANAGED_NON_CC_TABLE}"
```

path will be confusing in that context